sdk/eventhub/event-hubs/perf-tests.yml

```diff
@@ -0,0 +1,36 @@
+Service: eventhubs
+
+Project: sdk/eventhub/perf-tests/event-hubs
+
+PrimaryPackage: '@azure/event-hubs'
+
+PackageVersions:
+- '@azure/event-hubs': 5.8.0
```

Comment:
This file should also be taken care of by the release bot.

Whenever a new version of a package is released, it should update this file in the same PR.